src/style.css

```
@@ -0,0 +1,83 @@
+/* stylelint-disable length-zero-no-unit */
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: bold;
+  font-weight: 800;
+}
+
+* {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  color: #3d4e61;
+  font-weight: 400;
+}
```

```suggestion
* {
  box-sizing: border-box;
}
```
Don't reset all properties using * selector, better specify the exact tags you want to reset.
Leave only box-sizing under universal selector.
You can also set the font for the whole page and inherit where needed